contracts/src/token/erc721/mod.rs

```diff
@@ -1138,12 +1184,12 @@ mod tests {
         let balance = contract
             .balance_of(alice)
             .expect("should return the balance of Alice");
-        let one = U256::from(1);
-        assert!(balance >= one);
+
+        assert_eq!(initial_balance + U256::from(1), balance);
     }
 
     #[motsu::test]
-    fn error_when_reusing_token_id(contract: Erc721) {
+    fn mint_error_when_reusing_token_id(contract: Erc721) {
```

Comment:
I prefer more when those kind of tests looks like `error_when_*`.
As I remember previous name have been like `error_when_minting_twice` or smth but we changed it to what it is right now.
Wdt @alexfertel?

Reply:
Yeah, I agree that starting with `error` or `errors` is better.

I suggested `reusing_token_id` because "minting twice" is not something bad, so the test name gets confusing.

Maybe `error_when_minting_token_id_twice`?

Reply:
@bidzyyys let's also arrange other test names (e.g. transfer_from_error_when_transfer_to_invalid_receiver) like we discussed here